stdlib/macpath.pyi

```diff
@@ -25,35 +25,35 @@ if sys.version_info < (3, 8):
     if sys.version_info >= (3, 6):
         # Overloads are necessary to work around python/mypy#3644.
         @overload
-        def abspath(path: _PathLike[AnyStr]) -> AnyStr: ...
+        def abspath(path: PathLike[AnyStr]) -> AnyStr: ...
```

Comment:
I assume, these should be `AnyPath`, but I don't want to make this change in this PR.